packages/common/src/services/resizer.service.ts

```diff
@@ -251,7 +252,7 @@ export class ResizerService {
 
   /**
    * Provide the possibility to pause the resizer for some time, until user decides to re-enabled it later if he wish to.
-   * @param {boolean} isResizePaused are we pausing the resizer?
+   * @param {boolean} isResizePaused are we pausing the resizer?@C
```

Comment:
is that `@C` a typo?

Reply:
Nice catch. No idea how that got there...but I'll fix it.